custom_model_runner/datarobot_drum/drum/typeschema_validation.py

```diff
@@ -523,38 +523,22 @@ class SchemaValidator:
     actual validation on the respective dataframes.
     """
 
-    _DEFAULT_TYPE_SCHEMA_CODEDIR_PATH = os.path.abspath(
+    DEFAULT_TYPE_SCHEMA_CODEDIR_PATH = os.path.abspath(
```

Comment:
what is this and why is it here?

Reply:
as far as I can tell, this is a pointer to a random model-metadata.yaml file.   maybe a default one?  if so, this should be moved to the model_metadata.py file and be encapsultaed in `read_default_model_metadata`  or something like that.  we probably should never be explicitly referencing this file.